packages/engine/src/scene/systems/ShadowSystem.tsx

```diff
@@ -236,7 +237,7 @@ const DropShadowReactor = createQueryReactor([ShadowComponent], function DropSha
     if (sphere.radius > maxRadius) return
 
     const radius = Math.max(sphere.radius * 2, minRadius)
-    const center = groupComponent.value[0].worldToLocal(sphere.center)
+    const center = groupComponent.value[0].position
```

Comment:
this is a regression, as the point of using the bounding circle is for objects whose centers are significantly distant from the entity's transform. if the world/local space change fixes them appearing in the wrong location, then we can probably just use `sphere.center` directly here instead of `groupComponent.value[0].position` (equivalent to `TransformComponent position`)

Reply:
This was quite trouble some as it was relying on the groupComponents position updates. So instead of using worldToLocal I changed it to translate group position by that calculated bounding sphere center.